runelite-client/src/main/java/net/runelite/client/plugins/zalcano/ZalcanoOverlay.java

```diff
@@ -237,5 +268,17 @@ private void renderZalcanoAOE(Graphics2D graphics, int polySize, String text, Co
 		}
 	}
 
+	private void renderPlayerCount(Graphics2D graphics)
+	{
+		Rectangle rect = new Rectangle(client.getCanvasWidth() / 2 - 15, 0, 40, 25);
+		int[] xpoints = {rect.x, rect.x + rect.width, rect.x + rect.width, rect.x};
+		int[] ypoints = {rect.y, rect.y, rect.y + rect.height, rect.y + rect.height};
+
+		graphics.setFont(new Font("Comic Sans", Font.BOLD, 20));
+		OverlayUtil.renderFilledPolygon(graphics, rect, new Color(0, 0, 0));
```

Comment:
`Color.BLACK`

`Font` instances are reusable, and should be just like colors either configurable or static final constants